pkg/reconciler/claimbinding/reconciler.go

```diff
@@ -271,8 +267,6 @@ func (r *Reconciler) Reconcile(req reconcile.Request) (reconcile.Result, error)
 	// NOTE(negz): This method is well over our cyclomatic complexity goal.
 	// Be wary of adding additional complexity.
 
-	log.V(logging.Debug).Info("Reconciling", "controller", claimControllerName, "request", req)
```

Comment:
Do we not need this anymore? If I'm understanding the rest of the PR correctly, it looks like maybe these lines have been replaced by a single shared one? It does seem as though the shared one isn't logging the controller, but I could be missing something.

Reply:
I removed the package level logger from all reconcilers with the intention of plumbing new ones in, but as of the current draft of this PR I've only plumbed up the managed reconciler. So I imagine these will be replaced with a similar pattern to what's going on in the managed reconciler.